apps/platform-app/src/app/[locale]/(moderation)/_actions/mutations/approveProfile.ts

```diff
@@ -19,12 +20,14 @@ export const approveProfile = withSentry(async (profileId: string) => {
     state: PublishingState.APPROVED,
   })
 
-  const profileOwnerUsername = await findGithubUsernameByProfileId(profileId)
+  const profileOwnerUsername =
+    ((await findGithubUsernameByProfileId(profileId)) as string) ??
+    ((await findLinkedinUsernameByProfileId(profileId)) as string)
```

Comment:
Maybe wrap those two functions into one new helper function for better readability in the future?